### PR TITLE
Explain issue priorities

### PR DESCRIPTION
**Pull request by satnam6502**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,6 +163,15 @@ source.  We don't use it at Facebook and we have no plans to start.
 
 The blog [Optimizing Android bytecode with ReDex](https://code.facebook.com/posts/1480969635539475/optimizing-android-bytecode-with-redex) provides an overview of the Redex project.
 
+# Issues
+Issues on GitHub are assigned priorities which refelct their urgency and how soon they are
```
Comment (WizKid) [inline README.md]: reflect

Comment (satnam6502) [inline README.md]: Thanks: fixed, waiting for the ingestion process.

**Comment by satnam6502** [comment]
@facebook-github-bot update

**Comment by satnam6502** [comment]
@facebook-github-bot import

**Comment by satnam6502** [comment]
@facebook-github-bot import

**Comment by satnam6502** [comment]
@facebook-github-bot import

**Comment by facebook-github-bot-1** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/938180112947268/int_phab) to review.

**Comment by satnam6502** [comment]
@facebook-github-bot update

**Pull request closed without merging**